docker/dev.yml

```diff
@@ -35,3 +35,20 @@ services:
   scheduler:
     <<: *api-base
     command: ["python", "manage.py", "qcluster"]
+
+  update-listener:
```

Comment:
1) does this mean we can disable the cron tasks that daily sync the whole DB ? or should we have both running for a while ?
2) does redis send us changes from all flavors ?

Reply:
> does this mean we can disable the cron tasks that daily sync the whole DB ? or should we have both running for a while ?
I prefer having both running for a while

> does redis send us changes from all flavors ?
It should, but here again we should make sure it works well in prod before disabling the daily dump import